plugins/modules/jira.py

```diff
@@ -446,6 +458,15 @@
     operation: attach
     attachment:
       filename: topsecretreport.xlsx
+
+# Support for client certificate
+- name: Create an issue
+  community.general.jira:
+    uri: '{{ server }}'
+    username: '{{ user }}'
+    password: '{{ pass }}'
+    client_cert: '{{ path/to/client-cert }}'
+    client_key: '{{ path/to/client-key }}'
```

Comment:
If `client_cert` and `client_key` are mutually exclusive with `username` and `password`, why is this example showing all four?